src/main/java/org/osiam/Osiam.java

```diff
@@ -48,11 +55,32 @@
     );
 
     public static void main(String[] args) {
-        SpringApplication application = new SpringApplication(Osiam.class);
+        SpringApplication application = new SpringApplication();
         application.setDefaultProperties(DEFAULT_PROPERTIES);
+
+        String command = extractCommand(args);
+        if (command.equals("initHome")) {
+            application.setSources(Collections.<Object>singleton(InitHome.class));
+            application.setWebEnvironment(false);
+        } else {
+            application.setSources(Collections.<Object>singleton(Osiam.class));
+        }
+
         application.run(args);
     }
 
+    private static String extractCommand(String[] args) {
```

Comment:
`args` is a common abbreviation. But maybe in this function you could avoid using `arg` or `*Arg` and using `argument` instead?